AppCenter/AppCenter/Internals/Model/Util/MSUserDefaults.m

```diff
@@ -72,7 +77,11 @@ - (NSDictionary *)updateDictionary:(NSDictionary *)dict forKey:(NSString *)key {
 }
 
 - (BOOL)updateObject:(id)o forKey:(NSString *)key expiration:(float)expiration {
-  NSDictionary *update = [self updateDictionary:@{ @"v" : o } forKey:key expiration:expiration];
+  NSDictionary *update = [self updateDictionary:@{
+    @"v" : o
+  }
+                                         forKey:key
+                                     expiration:expiration];
```

Comment:
Formatting looks wrong here, should we really do it like this?

Reply:
It looks weird, I agree, but code formatting seems do do it like this.